src/enable.js

```diff
@@ -31,48 +33,76 @@ module.exports = function (flags) {
-      if (err) {
-        log.error('enable', err.message)
-        process.exit(2)
-      }
-
-      if (!data) {
-        return log.error('enable', story.error_no_data)
-      }
-
-      if (data.noChange) {
-        return log.error('enable', story.error_no_change(slug)) // TODO: We might not need this
-      }
-
-      if (data.ok) {
-        return console.log(story.enabled(slug))
-      }
-
-      if (data.statusCode === 400) {
-        log.error('enable', 'Couldn’t enable a repository with this slug.')
-        log.error('enable', 'If you want to try your free private repository make sure to grant the necessary rights by running ' + chalk.yellow('greenkeeper login --force --private'))
-        log.error('enable', 'You have to have a plan for more than one private repository. To verify run ' + chalk.yellow('greenkeeper whoami'))
-        log.error('enable', 'If you have just recently created this repository try running ' + chalk.yellow('greenkeeper sync'))
-        log.error('enable', 'You need admin access to a repository to enable it.')
-        log.error('enable', 'If you think this error really shouldn’t appear let us look into it with ' + chalk.yellow('greenkeeper support'))
-        process.exit(1)
-      } else if (data.statusCode === 403) {
-        log.error('enable', 'You need a paid greenkeeper.io subscription to enable private repositories\nYou can subscribe via ' + chalk.yellow('greenkeeper upgrade'))
-        process.exit(1)
-      }
-
-      log.error('enable', res.statusMessage + (res.body.message ? ': ' + res.body.message : ''))
-      process.exit(2)
-    })
+    enable(false)
+
+    function enable (isSynced) {
+      log.http('enable', 'Sending request')
+      request({
+        method: 'POST',
+        url: flags.api + 'packages',
+        json: true,
+        headers: {
+          Authorization: 'Bearer ' + flags.token
+        },
+        body: {slug: slug}
+      }, function (err, res, data) {
+        if (err) {
+          log.error('enable', err.message)
+          process.exit(2)
+        }
+
+        if (!data) {
+          return log.error('enable', story.error_no_data)
+        }
+
+        if (data.noChange) {
+          return log.error('enable', story.error_no_change(slug)) // TODO: We might not need this
+        }
+
+        if (data.ok) {
+          return console.log(story.enabled(slug))
+        }
+
+        if (data.statusCode === 400) {
+          if (!isSynced) {
+            log.verbose('enable', 'Repository not found. Starting a sync.')
+            log.info('enable', 'Synchronizing your repositories. This might take a while.')
+            var spin = spinner()
+            sync(flags, function (err, repos) {
+              clearInterval(spin)
+              if (err) {
+                log.error('enable', 'Synchronizing the repositories was not possible.')
+                return printFailedError()
+              }
+              isSynced = true
+              if (repos.indexOf(slug) > -1) {
```

Comment:
I'd prefer `_.contains` here, simply because reading the code makes more sense then :)